.github/workflows/python.yaml

```diff
@@ -49,3 +49,27 @@ jobs:
         run: |
           poetry config pypi-token.pypi $PYPI_TOKEN
           poetry publish --build
+  docs:
+    needs: test
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # fetch all branches, incl. gh-pages
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          pip install poetry
+          poetry install --extras "examples"
+      - name: Setup git author and version tag
+        run: |
+          git config --global user.name docs deploy
+          git config --global user.email docs@not.a.real.email.org
```

Comment:
AFAIK, it's good practice to use `*.local` TLDs for dummy emails, since `email.org` might exist outside of our control and be accidentally affected at some point down the line.

Reply:
TIL!